src/pages/get-involved.js

```diff
@@ -0,0 +1,222 @@
+import linkedInImage from "../images/LinkedIn-Logos/LI-In-Bug.png";
+import NewsletterSignUp from "../components/newsletter/newsletter-sign-up";
+
+const mainCss = css`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 1.5rem 0.75rem;
+  color: var(--midnight);
+
+  h1 {
+
+    font-size: 2rem;
+    line-height: 2.75rem;
+    font-weight: 600;
+  }
+
+  h2 {
+    font-size: 1.25rem;
+    font-weight: 600;
+    margin-bottom: 14px;
+  }
+
+  p {
+    font-size: 1rem;
+  }
+
+  .heading {
+    text-align: center;
+  }
+
+  .contact-items {
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    grid-row-gap: 1em;
+    grid-column-gap: 2.25em;
+    grid-auto-rows: minmax(100px, auto);
+    max-width: 1024px;
+  }
+
+  .contact-item {
+    margin-bottom: 36px;
+    p {
+      margin-bottom: 36px;
+    }
+  }
+
+  .more-contact {
+    p {
+      padding: 24px 0;
+      text-align: center;
+    }
+  }
+
+  .more-contact-item {
+    display: inline-block;
+    vertical-align: top;
+    width: 445px;
+    margin: 0 95px;
+
+    h2 {
+      margin-top: 26px;
+      margin-bottom: 4px ;
+    }
+
+    .linked-in {
+      display: inline-block;
+      width: 1em;
+      height: 1em;
+    }
+  }
+
+  .contact-panel {
+    box-shadow: 0 0 4px 0 rgba(0,0,0,0.2);
+  }
+
+  .heading {
+    margin-bottom: 95px;
+  }
+
+
+  a {
+    color: var(--midnight);
+    text-decoration: underline;
+  }
+
+  ${mq(BreakPoint.sm)} {
+    .contact-items {
+      grid-template-columns: repeat(2, 1fr);
+    }
+
+  }
+`;
+
+const headerCss = css`
+  font-weight: 600;
+
+  .header {
+    height: 12.5rem;
+  }
+
+  p {
+    color: var(--white);
+  }
+
+  .highlight {
+    font-size: 1.25rem;
+  }
+
+  h2 {
+    background-color: var(--yellow);
+    color: var(--midnight);
+    display: inline-block;
+    padding: 0.125rem;
+  }
+
+  .container {
+    padding: 0.75rem;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: center;
+    gap: 1rem;
+    height: 100%;
+  }
+
+  ${mq(BreakPoint.md)} {
+    .header {
+      padding: 0 2.5rem;
+      height: 16rem;
+    }
+    h2 {
+      font-size: 2rem;
+    }
+    .highlight {
+      font-size: 1.5rem;
+    }
+  }
+  ${mq(BreakPoint.lg)} {
+    .header {
+      padding: 0 10rem;
+      height: 18.75rem;
+    }
+  }
+`;
+
+const Header = () => (
+  <header css={headerCss}>
+    <HeroBackgroundImage className="header">
+      <div className="container">
+        <p>Home &gt; Get Involved</p>
+        <h2>Get Involved</h2>
+        <p className="highlight">&nbsp;</p>
+      </div>
+    </HeroBackgroundImage>
+  </header>)
+
+const Main = () => (
+  <main css={mainCss}>
+    <div className="heading">
+      <h1>Ways to get involved</h1>
+      <p>
+        Help us make the change. We have bold and ambitious plans and need help to execute them.
+      </p>
+    </div>
+    <div className="contact-items">
```

Comment:
Since this is a list, I'd be inclined to go for `ul.contact-items` and `li.contact-item`.